x-pack/platform/plugins/shared/streams_app/public/hooks/use_update_streams.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { useAbortController } from '@kbn/react-hooks';
+import { StreamUpsertRequest } from '@kbn/streams-schema';
+import { useCallback } from 'react';
+import { useKibana } from './use_kibana';
+
+export const useUpdateStreams = (name?: string) => {
```

Comment:
any way we can prevent the optional parameter here?

Reply:
We can probably have some definition null guard higher in the tree, and remove the optional in the component props and therefore in this hook. I'll take a look